src/events/message/messageDelete.ts

```diff
@@ -47,7 +47,7 @@ export default class extends Event {
 				content: message.content ? `**C:** ${Util.cleanContent(message.content, message.channel)}\n` : "",
 				reply: message.type === "REPLY" ? `**Reply to:** https://discord.com/channels/${message.reference?.guildId}/${message.reference?.channelId}/${message.reference?.messageId}\n` : "",
 				sticker: message.stickers.first() ? `**S:** ID: \`${message.stickers.first()?.id}\` | **Name:** ${message.stickers.first()?.name} | **Format:** ${message.stickers.first()?.format}\n` : "",
-				attachment: message.attachments.first() ? `**A**: ${message.attachments.first()?.proxyURL}\n` : "",
+				attachment: message.attachments.first() ? `**A**: ${message.attachments.map((attach: any) => `**${attach.name}**\n${attach.proxyURL}`).join("\n")}\n` : "",
```

Comment:
```suggestion
				attachment: message.attachments.first() ? `**A**: ${message.attachments.map((attach) => `**${attach.name}**\n${attach.proxyURL}`).join("\n")}\n` : "",
```
Same as the other one